Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/threshold/ThresholdedRandomCutForest.java

```diff
@@ -488,4 +804,347 @@ public void setLastExpectedPoint(double[] point) {
         return array == null ? null : Arrays.copyOf(array, array.length);
     }
 
+    /**
+     * maps a value shifted to the current mean or to a relative space
+     * 
+     * @param value     input value of dimension
+     * @param normalize flag that determines relative or shift
+     * @param deviation statistic
+     * @return the normalized value
+     */
+    double map(double value, boolean normalize, Deviation deviation) {
```

Comment:
I'm wondering if we could give these methods more descriptive names. Would itbe appropriate to call them `normalize` and `denormalize`?

Reply:
Yes :)